### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by hanzei**
The PR cleans up some code pieces by
- Checking previously underhanded errors
- Fixing the note that is created for a new ticket
- Formatting the code consistently

### CODE REVIEW

File: bindings.go [hunk 1]
```diff
@@ -4,10 +4,16 @@ import (
 	"net/http"
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/pkg/errors"
 )
 
 func bindings(w http.ResponseWriter, req *http.Request, creq *apps.CallRequest) {
-	user, _ := userFromContext(creq)
+	user, err := userFromContext(creq)
```
Comment (levb) [inline bindings.go:11]: 👍

File: create_note.go [hunk 2]
```diff
@@ -91,14 +90,18 @@ func createNote(w http.ResponseWriter, req *http.Request, creq *apps.CallRequest
 
 	// Post feedback to the channel, as the acting user
 	asUser := mmclient.AsActingUser(creq.Context)
-	asUser.CreatePost(&model.Post{
+	_, err = asUser.CreatePost(&model.Post{
```
Comment (levb) [inline create_note.go:93]: this one was purposely ignored

File: create_note_form.go [hunk 3]
```diff
@@ -35,7 +35,7 @@ func createNoteForm(w http.ResponseWriter, req *http.Request, creq *apps.CallReq
 		if content != "" {
 			content += "\n\n"
 		}
-		content = fmt.Sprintf("Note created from a message in Mattermost, %s:\n\n%s", permalink(creq), post.Message)
+		content += fmt.Sprintf("Note created from a message in Mattermost, %s:\n\n%s", permalink(creq), post.Message)
```
Comment (levb) [inline create_note_form.go:38]: 👍  thx!

**Review by levb — CHANGES_REQUESTED** [review]

File: create_note.go (hunk 2 above)
Comment (hanzei) [reply create_note.go:93]: I'm wondering why this is the case. Isn't it useful to return (and log) the error?

**Pull request merged**